src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "header">
+      <a href="#home" class="logo__link">
+        <img
+          class = "logo__img"
+          src="./images/logo.png"
+          alt="home"
```

Comment:
`alt` attribute describes the image, not the link, so "logo", as suggested by @BudnikOleksii is better than "home"

Still it is not critical - you may leave it as it is, just point it :)